loki/package.json

```diff
@@ -0,0 +1,104 @@
+{
+  "name": "@perses-dev/loki-plugin",
+  "version": "0.1.0",
+  "scripts": {
+    "dev": "rsbuild dev",
+    "build": "npm run build-mf && concurrently \"npm:build:*\"",
+    "build-mf": "rsbuild build",
+    "build:cjs": "swc ./src -d dist/lib/cjs --strip-leading-paths --config-file .cjs.swcrc",
+    "build:esm": "swc ./src -d dist/lib --strip-leading-paths --config-file .swcrc",
+    "build:types": "tsc --project tsconfig.build.json",
+    "test": "cross-env LC_ALL=C TZ=UTC jest",
+    "type-check": "tsc --noEmit"
+  },
+  "main": "lib/cjs/index.js",
+  "module": "lib/index.js",
+  "types": "lib/index.d.ts",
+  "devDependencies": {
+    "@module-federation/rsbuild-plugin": "^0.11.2",
+    "@rsbuild/core": "^1.2.19",
+    "@rsbuild/plugin-react": "^1.1.1",
+    "@types/react-dom": "^18.3.0",
+    "@types/jest": "^29.5.14",
+    "react": "^18.0.0",
+    "react-dom": "^18.0.0",
+    "react-router-dom": "^6.26.0",
+    "concurrently": "^9.1.2",
+    "jest": "^29.7.0",
+    "@swc/cli": "^0.6.0",
+    "@swc/core": "^1.7.10",
+    "@swc/jest": "^0.2.37",
+    "@testing-library/dom": "^10.4.0",
+    "@testing-library/jest-dom": "^6.6.3",
+    "@testing-library/react": "^16.1.0",
+    "@testing-library/user-event": "^13.5.0"
+  },
+  "peerDependencies": {
+    "@emotion/react": "^11.7.1",
+    "@emotion/styled": "^11.6.0",
+    "@hookform/resolvers": "^3.2.0",
+    "@perses-dev/components": "^0.51.0-beta.1",
+    "@perses-dev/core": "^0.51.0-beta.1",
+    "@perses-dev/dashboards": "^0.51.0-beta.1",
+    "@perses-dev/explore": "^0.51.0-beta.1",
+    "@perses-dev/plugin-system": "^0.51.0-beta.1",
+    "@tanstack/react-query": "^4.39.1",
+    "date-fns": "^4.1.0",
+    "date-fns-tz": "^3.2.0",
+    "echarts": "5.5.0",
+    "immer": "^10.1.1",
+    "lodash": "^4.17.21",
+    "react": "^17.0.2 || ^18.0.0",
+    "react-dom": "^17.0.2 || ^18.0.0",
+    "react-hook-form": "^7.52.2",
+    "use-resize-observer": "^9.0.0"
+  },
+  "files": [
+    "lib/**/*",
+    "__mf/**/*",
+    "mf-manifest.json",
+    "mf-stats.json"
+  ],
+  "perses": {
+    "moduleName": "Loki",
+    "schemasPath": "schemas",
+    "plugins": [
+      {
+        "kind": "Datasource",
+        "spec": {
+          "display": {
+            "name": "Loki Datasource"
+          },
+          "name": "LokiDatasource"
+        }
+      },
+      {
+        "kind": "TimeSeriesQuery",
+        "spec": {
+          "display": {
+            "name": "Loki Time Series Query"
+          },
+          "name": "LokiTimeSeriesQuery"
+        }
+      },
+      {
+        "kind": "LogQuery",
+        "spec": {
+          "display": {
+            "name": "Loki Log Query"
+          },
+          "name": "LokiLogQuery"
+        }
+      },
+      {
+        "kind": "Panel",
```

Comment:
Does it make sense this panel is an independent plugin instead of being related to the `Loki` datasource ?

Reply:
The question is, how much (are we confident that) this panel is agnostic to Loki specificities & could be compatible with other future log data sources?

Reply:
> Aligning both datasources would require adjusting two PRs at once, which feels like too much for now. I think we should properly discuss the direction first, and I’m open for feedback.

Well I think having in another PR the TableLogs panel would not hurt and it will open the discussion. Then afterwards we can align both PRs or plugins if the PRs are already merged.

Reply:
yes that would be acceptable as long as the new panel is named `LogsTable` to avoid a breaking change when the panel contained in this plugin will be removed.